ci/test_python_singlegpu.sh

```diff
@@ -19,6 +19,21 @@ pytest \
   --cov=cuml \
   --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cuml-coverage.xml" \
   --cov-report=term \
+  -m "not memleak" \
+  .
+
+  rapids-logger "memory leak pytests..."
+
+pytest \
+  --numprocesses=1 \
+  --ignore=dask \
+  --cache-clear \
+  --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml.xml" \
```

Comment:
```suggestion
  --junitxml="${RAPIDS_TESTS_DIR}/junit-cuml-memleak.xml" \
```

Looks like this file name still needs to be updated.

It should different from the `xml` file name from the previous `pytest` invocation.